extensions-contrib/druid-iceberg-extensions/pom.xml

```diff
@@ -271,6 +272,22 @@
       <artifactId>iceberg-hive-metastore</artifactId>
       <version>${iceberg.core.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.iceberg</groupId>
+      <artifactId>iceberg-aws</artifactId>
+      <version>${iceberg.core.version}</version>
+    </dependency>
+<!--    GlueCatalog class-->
+    <dependency>
+      <groupId>software.amazon.awssdk</groupId>
+      <artifactId>bundle</artifactId>
+      <version>${awssdk.version}</version>
+    </dependency>
+<!--  org/apache/avro/LogicalType load-->
+    <dependency>
```

Comment:
Not sure! But I got error for no class definition while reading icebert data file

Reply:
```
java.lang.NoClassDefFoundError: org/apache/avro/LogicalType iceberg
```

Reply:
Instead of adding a new entry here, you can remove the apache avro dependency exclusion from the `hadoop-common` dependency